examples/ping-pong/src/main.rs

```diff
@@ -129,18 +134,17 @@ async fn main() -> ! {
         .register_handler(Arc::new(AsyncMutex::new(radio_handler)))
         .expect("Could not register handler");
 
-    let network = NetworkName::from_string(&config.graphcast_network);
+    let network = NetworkName::from_string("goerli");
```

Comment:
perhaps utilize config `GraphcastAgentConfig graphcast_namespace`?

Reply:
I am a bit confused here - `NetworkName` is the chains we support, but `graphcast_namespace` is either `testnet` or `mainnet`?

Reply:
oh hmm, so you have set `graphcast_namespace: graphcast_network.map(|s| s)`, which should be only `testnet` or `mainnet`, but we would want a Ethereum network here. You are right to hardcode "goerli" here